lib/src/identity_codec.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:convert';
+
+class _IdentityConverter<T> extends Converter<T, T> {
+  @override
```

Comment:
We don't use `@override` in SDK sources.
There are more below.

Reply:
Cool, I thought it was wide spread policy to write `@override` :)

I'm happy not to do that, hehe